src/components/WalletDropdown/MiniPortfolio/Activity/ActivityRow.tsx

```diff
@@ -45,7 +49,16 @@ export function ActivityRow({
       <PortfolioRow
         left={
           <Column>
-            <PortfolioLogo chainId={chainId} currencies={currencies} images={logos} accountAddress={otherAccount} />
+            {definedCurrencies?.length > 0 ? (
+              <PortfolioLogo
```

Comment:
`PortfolioLogo` already has logic to deal with undefined currencies. I think this file can be left as-is.